core/src/storage/impls/storage_db/snapshot_db_sqlite.rs

```diff
@@ -12,6 +12,7 @@ pub struct SnapshotDbSqlite {
 pub struct SnapshotDbStatements {
     kvdb_statements: KvdbSqliteStatements,
     mpt_statements: KvdbSqliteStatements,
+    temp_insert_statements: KvdbSqliteStatements,
```

Comment:
Replace to delta_mpt_set_keys_statements.

Leave a TODO for delta_mpt_delete_keys_statements.